### DISCUSSION TITLE
How goes the effort for the next version? :)

### DISCUSSION THREAD

**Discussion by philstopford** [CineEncoder/cine-encoder#55] [General]
Just checking in - this is a fantastically useful tool

**Comment by CineEncoder** [comment]
Hello!
The most difficult tasks remain for the release of version 3.5.5, I will have to think about how to do them, the last few months have been a lot of work, but I think I will do it.
I settled on the "Fix the output of error messages" task. It seems to work, but I still need to check with several examples specifically for files with errors.
I'm thinking of transferring some tasks to release 3.6, for example, "Add netherlands translation" and possibly "Add the ability to configure independent encoding of audio tracks", since the release of 3.5.5 was going to be released on December 15th. I'm thinking of postponing the release to February.
